config/settings/development.yml

```diff
@@ -9,4 +9,6 @@ forms_api:
   auth_key: changeme
 
 # Use mock authentication
+# in development use 'mock_gds_sso' to always be logged in or 'developer' to use a
+# mock flow through omniauth
 auth_provider: mock_gds_sso
```

Comment:
It's very hard to test this change without matching production more, hence adding the developer provider.

Reply:
Nice!